lib/spreedly/base.ex

```diff
@@ -38,29 +38,28 @@ defmodule Spreedly.Base do
       Application.get_env(:spreedly, :base_url)
 
   """
-  @spec process_url(path :: binary) :: binary
```

Comment:
Was this typespec failing for some reason?

Reply:
Yes, we were receiving a `has overlapping domains; such contracts are currently unsupported and are simply ignored` error.

Reply:
Ahhh because `HTTPoison` also has a `process_url` function?

Reply:
Yup. We overwrite the function, but I believe the spec was changed.